query-coordinator/src/main/scala/com/socrata/querycoordinator/resources/QueryService.scala

```diff
@@ -183,6 +183,10 @@ trait QueryService {
     GatewayTimeout
   }
 
+  def ranOutOfRetriesResponse: HttpResponse = {
+    internalServerError
```

Comment:
I considered it, but it's not really a _timeout_ (e.g., this can be caused by connections being refused outright).  I can make it be so if you really think it should be, though.